generator/config/expression/dayOfMonth.yaml

```diff
@@ -22,3 +22,14 @@ arguments:
         optional: true
         description: |
             The timezone of the operation result. timezone must be a valid expression that resolves to a string formatted as either an Olson Timezone Identifier or a UTC Offset. If no timezone is provided, the result is displayed in UTC.
+tests:
+    -
+        name: 'Example'
+        link: 'https://www.mongodb.com/docs/manual/reference/operator/aggregation/dayOfMonth/#example'
+        pipeline:
+            -
+                $project:
+                    day:
+                        # $dayOfMonth: '$date'
+                        $dayOfMonth:
+                            date: '$date'
```

Comment:
The [example](https://www.mongodb.com/docs/manual/reference/operator/aggregation/dayOfMonth/#example) is the same for all this date part operators. So I picked only the tested operator for each.